framework_tvb/tvb/adapters/creators/allen_creator.py

```diff
@@ -668,12 +735,12 @@ def mouse_brain_visualizer(vol, order, key_ord, unique_parents, unique_grandpare
 # the relation between the different reference system is: x1=z2, y1=x2, z1=y2
 def rotate_reference(allen):
     # first rotation in order to obtain: x1=x2, y1=z2, z1=y2
-    vol_trans = np.zeros((allen.shape[0], allen.shape[2], allen.shape[1]), dtype=float)
+    vol_trans = np.zeros((allen.shape[0], allen.shape[2], allen.shape[1]), dtype=int)
```

Comment:
the change was made here and in the review below from float to int because otherwise we would get a error on RegionVolumeMapping.array_data that says : "can't be set to an array of dtype float64" the fix to int solves this problem